packages/common/src/items/updateResource.ts

```diff
@@ -0,0 +1,71 @@
+import { IUserRequestOptions } from "@esri/arcgis-rest-auth";
```

Comment:
I know we'd discussed having `updateResource` as a separate function, but the code is essentially identical to `addResource` that I'm wondering how we can DRY it up - maybe adding an optional flag argument (`update`) to `addResource` (set to `false` by default) which conditionally changes which `arcgis-rest-portal` function is called. Then from this function we'd simply call `addResource` with the `update` flag set to true.